nni/retiarii/execution/api.py

```diff
@@ -21,9 +22,10 @@ def set_execution_engine(engine: AbstractExecutionEngine) -> None:
     if _execution_engine is None:
         _execution_engine = engine
     else:
-        raise RuntimeError('Execution engine is already set. '
-                           'You should avoid instantiating RetiariiExperiment twice in one process. '
-                           'If you are running in a Jupyter notebook, please restart the kernel.')
+        warnings.warn('Execution engine is already set. '
```

Comment:
Have you tested the behavior of running a notebook cell twice? Does it work, or not?

Reply:
checked, and fixed bugs